### DISCUSSION TITLE
建议新增主动探测字段

### DISCUSSION THREAD

**Discussion by jiuqianyuan** [gkd-kit/gkd#460] [Ideas]
目前只有很少量的app需要主动探测，可以新增一个主动探测的字段`activeMatch`，由用户手动控制，避免不需要主动的app循环探测增加耗电。
可将逻辑设为,大佬看看是否有必要

```
if ( activeMatch ){
  if(t - matchChangedTime < matchLimitTime || t - lastTriggerTime < 5_000){
      循环探测
  }
}
```

https://github.com/gkd-kit/gkd/blob/b24e68ec8fdf5cbce464bb380383c3a4d4319cbd/app/src/main/kotlin/li/songe/gkd/service/GkdAbService.kt#L188-L195

**Comment by lisonge** [comment]
https://gkd.li/api/interfaces/RawCommonProps.html#forcedtime
